inSociety/Controllers/SendRequestViewController.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  ProfileViewController.swift
+//  inSociety
+//
+//  Created by Aleksey Libin on 18.10.2022.
+//
+
+import UIKit
+import SDWebImage
+
+class SendRequestViewController: UIViewController {
+    
+    private let scrollView = UIScrollView()
+    
+    private let containerView = UIView()
+    private let imageView = UIImageView()
+    private let nameLabel = UILabel()
+    private let descriptionLabel = UILabel()
+    private let sendMessageTextField = SendMessageTextField()
+    
+    
+    private let user: UserModel
+    
+    init(user: UserModel) {
+        self.user = user
+        
+        self.nameLabel.text = user.userName
+        self.descriptionLabel.text = user.description
+        self.imageView.sd_setImage(with: URL(string: user.userAvatarString))
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupViews()
+        
+        if let button = sendMessageTextField.rightView as? UIButton {
+            button.addTarget(self, action: #selector(sendButtonPressed), for: .touchUpInside)
+        }
+        
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        
+        scrollView.contentSize = view.frame.size
+        
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+
+//MARK: - Button realization
+extension SendRequestViewController {
+    @objc private func sendButtonPressed() {
+        guard
+            let message = sendMessageTextField.text,
+            !message.isEmpty
+        else { return }
+        
+        self.dismiss(animated: true) {
```

Comment:
its better to start loader and process it, then dismiss, or dismiss and do this in vc where it navigates, but on dismiss completion - bad idea